docs/tasks/administer-cluster/securing-a-cluster.md

```diff
@@ -0,0 +1,196 @@
+---
+assignees:
+- smarterclayton
+title: Securing a Cluster
+redirect_from:
```

Comment:
This also looks like a copy/paste error. Since these docs are new, there doesn't need to be any redirect_from entries.